### PR TITLE
Adds delete functionality to user tokens list

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This addresses #7434 

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME
 - UI

##### ADDITIONAL INFORMATION
Screen Shot 2020-07-26 at 4 13 33 PM

### CODE REVIEW

File: awx/ui_next/src/screens/User/UserTokenList/UserTokenList.jsx [hunk 1]
```diff
@@ -54,84 +57,127 @@ function UserTokenList({ i18n }) {
...
+        }
+      />
+      {deletionError && (
+        <AlertModal
+          isOpen={deletionError}
+          variant="danger"
+          title={i18n._(t`Error!`)}
+          onClose={clearDeletionError}
+        >
+          {i18n._(t`Failed to delete one or more schedules.`)}
```
Comment (nixocio) [inline awx/ui_next/src/screens/User/UserTokenList/UserTokenList.jsx]:
```suggestion
          {i18n._(t`Failed to delete one or more user tokens.`)}
```

**Review by nixocio — APPROVED** [review]
I got this code locally. Changes seem to be working as expected.

**Review by jlmitch5 — APPROVED** [review]
looks good other than string change kersom mentioned

**Review by dsesami — APPROVED** [review]

**Comment by dsesami** [comment]
@AlexSCorey if you can adjust the lint requirements and merge

**Comment by ryanpetrello** [comment]
recheck

**Pull request merged**